esphome/components/web_server/__init__.py

```diff
@@ -69,3 +71,5 @@ async def to_code(config):
         path = CORE.relative_config_path(config[CONF_JS_INCLUDE])
         with open(file=path, mode="r", encoding="utf-8") as myfile:
             cg.add(var.set_js_include(myfile.read()))
+    if CONF_INCLUDE_INTERNAL in config:
+        cg.add(var.set_include_internal(config[CONF_INCLUDE_INTERNAL]))
```

Comment:
So that this can be unconditional (probably better to move it up as well):
```suggestion
    cg.add(var.set_include_internal(config[CONF_INCLUDE_INTERNAL]))
```